src/library/scala/math/package.scala

```diff
@@ -315,13 +356,83 @@ package object math {
   /** Returns the size of an ulp of the given `Double` value.
     * @group ulp
     */
-  def ulp(x: Double): Double = java.lang.Math.ulp(x)
+  def ulp(d: Double): Double = java.lang.Math.ulp(d)
```

Comment:
While `d` and `f` is now consistent with the naming convention used in the added methods which is based on the names used in Java I don't see this as an improvement because although this acts as a mnemonic to the parameter type it does not remind the user of the role of the parameter. Let the IDE and compiler remind us of the type; Use the name to describe the parameter along a different axis.

Although it's tricky to pick a name based on the role of the parameter here I would suggest `middle` based on the description of `ulp` in the Java docs. Even just reverting to `x` is better than switching to `d` and `f`.

WDYT?